.github/workflows/ci_cron_daily.yml

```diff
@@ -0,0 +1,44 @@
+name: Daily cron
+
+# This is daily cron on astropy dev but not here because
+# release branch is low traffic. But we keep the file as close
+# to dev version as possible for ease of backport.
+on:
+  push:
+    branches:
+      - v4.2.x
+  pull_request:
+    branches:
+      - v4.2.x
+
+jobs:
+  tests:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+    if: github.repository == 'astropy/astropy'
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+
+          - name: Bundling with pyinstaller
+            os: ubuntu-latest
+            python: 3.8
+            toxenv: pyinstaller
```

Comment:
Maybe move this to the main workflow as well ? It seems useless to run it daily as there are not commits every day pushed to those release branches.

Reply:
All the cron files are actually push events now. I added comments to explain that I keep the files as-is for ease of backporting (in case we do a 4.2.x release).